3p/recaptcha.js

```diff
@@ -76,8 +73,9 @@ init();
 
 /**
  * Main function called by the recaptcha bootstrap frame
+ * @param {string} recaptchaApiBaseUrl
  */
-export function initRecaptcha() {
+export function initRecaptcha(recaptchaApiBaseUrl) {
```

Comment:
can we give it a default value instead. `recaptchaApiBaseUrl= RECAPTCHA_API_URL`